helm-charts/namespace-cleaner/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Delete expired namespace
```

Comment:
put a link back to github readme in the description?

Reply:
added home and sources to this yaml. They are for links